pkg/engine/armvariables_test.go

```diff
@@ -143,8 +143,10 @@ func TestK8sVars(t *testing.T) {
 		"apiVersionManagedIdentity":      "2015-08-31-preview",
 		"apiVersionNetwork":              "2018-08-01",
 		"apiVersionStorage":              "2018-07-01",
+		"applicationInsightsKey":         "906ba2b3-f3b7-4cbe-ba9f-885d6982de43", // should be DefaultApplicationInsightsKey,
```

Comment:
@devigned I can't seem to DefaultApplicationInsightsKey from the api pkg here. Is there something i need to do to refresh to version of the package that is getting imported?

Reply:
taking a look...

Reply:
I guess I'm not sure what the problem is. Looks like this unit test is passing. Can you explain a bit more or add a failing test?